src/analyses/README.md

```diff
@@ -69,7 +69,19 @@ To be documented.
 
 \subsection analyses-escape Escape analysis (escape_analysist)
 
-To be documented.
+Intended to implement `__CPROVER_cleanup` instructions, which say that a particular
```

Comment:
Not that it would be all that great, but a pointer to https://en.wikipedia.org/wiki/Escape_analysis might still be helpful.

Reply:
done